mne/io/eyelink/_utils.py

```diff
@@ -0,0 +1,130 @@
+"""Helper functions for reading eyelink ASCII files."""
+# Authors: Scott Huberty <seh33@uw.edu>
+# License: BSD-3-Clause
+
+import re
+import numpy as np
+
+from .calibration import Calibration, Calibrations
+
+
+def _find_recording_start(lines):
+    """Return the first START line in an eyelink ASCII file.
```

Comment:
Do we want to / have any obligations to use the correct product names here? Then it would be EyeLink.